pyanaconda/ui/gui/spokes/software.py

```diff
@@ -396,7 +396,7 @@ def refresh(self):
             elif firstEnvironment:  # manual installation
                 # for manual installs that don't have a default provided by the install class
                 # just tick the first radio button and select the first environment
-                if not self.environment_valid:
+                if not self.environment_valid or self.environment is None:
```

Comment:
Am I wrong or does this mean that ``self.environment_valid`` is ``True`` for ``self.environment`` being ``None``? That seems at least really unfortunate.